src/modules/navigator/rtl.cpp

```diff
@@ -622,6 +607,8 @@ void RTL::init_rtl_mission_type()
 	default:
 		break;
 	}
+
+	_rtl_mission_type_handle->initialize();
```

Comment:
There's no guarantee `_rtl_mission_type_handle` exists. Only use pointers if really necessary 👀

Reply:
I fixed this one but there might be more corner cases. It's playing with fire on a drone 😬

Reply:
c8ade38